CMakeLists.txt

```diff
@@ -73,6 +73,16 @@ if (CMAKE_BUILD_TYPE STREQUAL "")
     set( CMAKE_BUILD_TYPE "RelWithDebInfo" )
 endif()
 
+option(ENABLE_WSTRING "Enable std::wstring support" ON)
+if (NOT ENABLE_WSTRING)
+    add_definitions(-DPOCO_NO_WSTRING)
+endif()
+
+option(ENABLE_FPENVIRONMENT "Enable floating-point support" ON)
```

Comment:
Change to `POCO_ENABLE_FPENVIRONMENT`, I will merge next time the pull request #1437 so POCO properties are in one group in cmake ....